org.freecad.FreeCAD.yaml

```diff
@@ -1,17 +1,14 @@
 app-id: org.freecad.FreeCAD
-app-id: org.freecad.FreeCAD
-default-branch: beta
 runtime: org.kde.Platform
-runtime-version: '6.7'
+runtime-version: '6.9'
 sdk: org.kde.Sdk
 command: FreeCAD
-desktop-file-name-prefix: "(beta) "
```

Comment:
this should not be removed.

Reply:
Because it gets removed by whichever method you used to resynchronize the changes, whether it's merge or cherry-picking the patches.